### PR TITLE
remove parameter db_type

### PR DESCRIPTION
**Pull request by sebcbi1**
remove parameter db_type to allow custom keybindings outside plugin

for example : 
vim.keymap.set("n", "<leader>sl", require('sqlrun').execute_current_line, { desc = "sql execute currentline" })

### CODE REVIEW

**Comment by mrvaita** [comment]
Hello, thanks a lot for this PR. After I merged #1 and #2 as expected there are conflicts. Please feel free to fix them and I can merge it.

**Comment by sebcbi1** [comment]
Your welcome . i was thinking these days trying to make a similar database plugin based on cli tools . i fall upon you post on reddit . your project meets my requirements / ideas . so i'll be happy to contribute

File: lua/sqlrun.lua [hunk 1]
```diff
@@ -175,12 +189,9 @@ function SqlRun.setup(config)
...
     local map_opts = { noremap = true, silent = true, nowait = true }
-    local execute_buffer_cmd = ":lua require('sqlrun').execute_buffer(\"%s\")<CR>"
-    local execute_selection_cmd = ":lua require('sqlrun').execute_selection(\"%s\")<CR>"
-    local execute_line_cmd = ":lua require('sqlrun').execute_current_line(\"%s\")<CR>"
-    vim.api.nvim_buf_set_keymap(0, "n", "<leader>q", string.format(execute_buffer_cmd, db_type), map_opts)
-    vim.api.nvim_buf_set_keymap(0, "v", "<leader>q", string.format(execute_selection_cmd, db_type), map_opts)
-    vim.api.nvim_buf_set_keymap(0, "n", "<leader>l", string.format(execute_line_cmd, db_type), map_opts)
+    vim.api.nvim_buf_set_keymap(0, "n", "<leader>q", ":lua require('sqlrun').execute_buffer<CR>", map_opts)
+    vim.api.nvim_buf_set_keymap(0, "v", "<leader>q", ":lua require('sqlrun').execute_selection<CR>", map_opts)
+    vim.api.nvim_buf_set_keymap(0, "n", "<leader>l", ":lua require('sqlrun').execute_line_cmd", map_opts)
```
Comment (mrvaita) [inline lua/sqlrun.lua]: Could you please change lines 192, 193 and 194 as
```lua
      vim.api.nvim_buf_set_keymap(0, "n", "<leader>q", ":lua require('sqlrun').execute_buffer()<CR>", map_opts)
      vim.api.nvim_buf_set_keymap(0, "v", "<leader>q", ":lua require('sqlrun').execute_selection()<CR>", map_opts)
      vim.api.nvim_buf_set_keymap(0, "n", "<leader>l", ":lua require('sqlrun').execute_current_line()<CR>", map_opts)
```
The function calls without parenthesis do not work on my local.

File: lua/util.lua [hunk 2]
```diff
@@ -166,7 +166,7 @@ function M.get_connection_string(server, port, user, password, db_name, binary,
     end
   end
 
-  return { command = connection_string, database = db_name }
+  return { command = connection_string, database = db_name , db_type = db_type }
```
Comment (mrvaita) [inline lua/util.lua]: could you please remove the space between `db_name` and `,`?

**Review by mrvaita — COMMENTED** [review]
Thank you again. I added a couple of comments to your changes. Overall I think it is a good idea to have the possibility to have custom key bindings to execute the queries.

**Comment by sebcbi1** [comment]
sorry for that , i'm quite new with lua script

**Comment by mrvaita** [comment]
No worries, I am not the most expert in lua script as well. Now it looks good

**Pull request merged**